packages/airnode-validator/package.json

```diff
@@ -6,8 +6,8 @@
   "main": "./dist/src/index.js",
   "types": "./dist/src/index",
   "bin": {
-    "api3-validator": "./dist/bin/validator.js",
-    "api3-convertor": "./dist/bin/convertor.js"
+    "validator": "./dist/bin/validator.js",
+    "convertor": "./dist/bin/convertor.js"
```

Comment:
```suggestion
    "airnode-validator": "./dist/bin/validator.js",
    "airnode-convertor": "./dist/bin/convertor.js"
```

Reply:
Do we need this? This is only visible to the user if running with `npx`. You invoke this like this:

```sh
npx -p @api3/airnode-validator -c validator
```

so I find the prefix for this unnecessary.

Reply:
That's not true. Based on this the package is packed and binaries are created. So this is to avoid having a command in the system that's called just `validator`.

Reply:
Yeah, you're right. I forgot about the global install